data/FilterList.json

```diff
@@ -2487,7 +2487,7 @@
   {
     "id": 216,
     "chatUrl": "https://discord.me/polskiefiltry",
-    "description": "Filters that hide and block social network elements, i.e. widgets, pop-ups and other social elements on Polish websites.",
+    "description": "Filters that hide and block social network elements, i.e. widgets, pop-ups and other social elements on Polish websites. Fanboy's Social Blocking List is required for proper operation.",
```

Comment:
We could also add an entry [here](https://github.com/collinbarrett/FilterLists/blob/master/data/Dependent.json) that marks the dependency. Dependencies aren't show in the UI yet, but they are on the TODO list: #465